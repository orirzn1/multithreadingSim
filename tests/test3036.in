L13
S7
O8
N5

T0
I@0x00000000
STORE $2, $1, 0x3
ADD $3, $1, $7
STORE $7, $1, $2
SUB $3, $0, $1
STORE $4, $7, $2
ADDI $2, $2, 0x2
ADDI $3, $2, 0x1
SUBI $2, $1, 0x1
SUB $1, $1, $1
HALT $0

T1
I@0x00000000
SUB $6, $6, $7
ADD $2, $2, $6
ADDI $5, $5, 0x2
SUB $2, $0, $5
HALT $0

T2
I@0x00000000
SUBI $7, $6, 0x4
LOAD $4, $4, $3
SUB $3, $0, $7
ADD $2, $0, $1
LOAD $4, $4, 0x4
STORE $1, $1, $4
SUB $5, $4, $6
HALT $0

T3
I@0x00000000
SUB $7, $7, $2
LOAD $2, $3, $6
ADDI $6, $6, 0x4
ADD $1, $5, $6
SUB $1, $2, $5
HALT $0

T4
I@0x00000000
LOAD $0, $3, $0
SUBI $5, $0, 0x1
STORE $4, $0, 0x2
SUB $4, $5, $7
LOAD $1, $2, 0x3
LOAD $7, $5, $3
HALT $0

D@0x0
0x2
0x6
0x1
0x1
0x5
0xA
0x9
0x8
0x4
0x9
