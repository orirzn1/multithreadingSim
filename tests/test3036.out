
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x3	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFE	R3 = 0x0	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x4	R4 = 0x6	R5 = 0x6	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 4.472222

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x3	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFE	R3 = 0x0	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x4	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x4	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.694444

